db/migrate/20220818194413_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postsCounter 
+      t.timestamps
```

Comment:
- [ ] [OPTIONAL] Let's remember that Ruby's naming convention should be observed when writing variables or data attributes for our databases. `postsCounter` should be written as `posts_counter`.

Reply:
Thank you @elisha2kyakpo1 I will correct that one